includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -696,6 +699,11 @@ private function getCommunity() {
 		return $data;
 	}
 
+	private function isLocalSitemapAvailable() {
+		$default = true; // $wgEnableLocalSitemapPageExt = true; in CommonSettings
```

Comment:
can you take this value from `$wgEnableLocalSitemapPageExt` instead of hardcoding it?